### ISSUE TITLE
Breadcrumbs should have a nav wrapper in the example code to be WCAG compliant

### ISSUE THREAD

**Issue by boye-s** [SwedbankPay/design.swedbankpay.com#101]
According to WCAG 2.0 breadcrumbs should be surrounded by a `nav` element.
This should be a part of the design guide example for breadcrumbs.
Ref: https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html

**Comment by eskilsen** [comment]
Fixed 🚀
